Source/Microsoft.Teams.Apps.FAQPlusPlus/Cards/SmeFeedbackCard.cs

```diff
@@ -136,7 +145,7 @@ private static string GetRatingDisplayText(string rating)
                 throw new ArgumentException($"{rating} is not a valid rating value", nameof(rating));
             }
 
-            return Strings.ResourceManager.GetString($"{rating}RatingText", CultureInfo.InvariantCulture);
+            return Strings.ResourceManager.GetString($"{rating}RatingText", CultureInfo.CurrentCulture);
```

Comment:
Why CurrentCulture? They're both strings, so this is effectively just string concatenation: `rating + "RatingText"`

Reply:
Actually we have some stored value in Resources files. 
Like this 
- [HelpfulRatingText](https://github.com/OfficeDev/microsoft-teams-apps-faqplus/blob/78de32393d2d3539f87623a80818516ae9646058/Source/Microsoft.Teams.Apps.FAQPlusPlus/Properties/Strings.resx#L374)
- [NeedsImprovementRatingText](https://github.com/OfficeDev/microsoft-teams-apps-faqplus/blob/78de32393d2d3539f87623a80818516ae9646058/Source/Microsoft.Teams.Apps.FAQPlusPlus/Properties/Strings.resx#L378)
- [NotHelpfulRatingText](https://github.com/OfficeDev/microsoft-teams-apps-faqplus/blob/78de32393d2d3539f87623a80818516ae9646058/Source/Microsoft.Teams.Apps.FAQPlusPlus/Properties/Strings.resx#L402)

In argument **_rating_** we have value as Helpful, NeedsImprovement and NotHelpfulRating.
To show this text translated it is changed it to CurrentCulture.

Reply:
Oh doh you're right, that was an argument to GetString, not the formatting operation.